plugins/ipam/static/main.go

```diff
@@ -193,6 +193,10 @@ func LoadIPAMConfig(bytes []byte, envArgs string) (*IPAMConfig, string, error) {
 		// args IP overwrites IP, so clear IPAM Config
 		n.IPAM.Addresses = make([]Address, 0, len(n.Args.A.IPs))
 		for _, addr := range n.Args.A.IPs {
+			_, _, err := net.ParseCIDR(addr)
```

Comment:
the  string-format IP(or CIDR) will be parsed later, so why we check this twice?

Reply:
This was the way I found of indicating which parameter was the offending one (the string format CIDR validation will be performed for cni args / ipam config later on, and its impossible to know the parameter name by then).

But I agree with you - IMO, just saying we expect CIDR notation should be enough.

Reply:
OK, since the parse and validation already done here, could we append a parsed `Address` instead of a single string to avoid this to be parsed later?

Reply:
I am not entirely fond of how it looks, but have provided a new commit w/ this change. 

Let me know if that's OK.